src/web/components/CardHeadline.tsx

```diff
@@ -131,7 +131,9 @@ export const CardHeadline = ({
 
 				<span
 					className={css`
-						color: ${palette.text.cardHeadline};
+						color: ${isFullCardImage
+							? neutral[100]
+							: palette.text.cardHeadline};
```

Comment:
Can we make this change in `decidePalette`? Sorry to nag and I appreciate that we originally started with `isFullCardImage` and the timing for introducing `palette` at the same time as you're doing this work isn't ideal. But I'm actually all over the codebase doing the same thing and I'm quite keen for future changes to adopt this pattern.

In `Carousel.tsx` I added the line

```
if (isFullCardImage) trails = convertToImmersive(trails);
```

which means that each `trail` after that point has `format.display` set to `Display.Immersive` and this is the value used to decide the `palette` property. If you then look at `decidePalette` there's a line

```
if (format.display === Display.Immersive) return WHITE;
```

which is where this decision about the headline colour to use for immersive cards.

Reply:
By the way, I _think_ this is already set to white so I'm not sure if this change does anything?

Reply:
Ah yes. I added that line in before running storybook so I added it in and didn't realise it was superfluous. I've removed it now which makes it much neater